tests/test_controller.py

```diff
@@ -422,7 +425,11 @@ def mock_for_library(cls, incoming_library):
 
         ConfigurationSetting.sitewide(
             self._db, Configuration.AUTHENTICATION_DOCUMENT_CACHE_TIME
-        ).value = "0"
+        ).value = "60"
```

Comment:
I'd add a comment that these settings are seconds where the Configuration.AUTHENTICATION_DOCUMENT_CACHE_TIME value is in MS - just for clarity when reviewing test output.

Reply:
Where are you getting the milliseconds idea? The default value for AUTHENTICATION_DOCUMENT_CACHE_TIME is 3600 (which I'm about to change in a follow-up).